### ISSUE TITLE
WCAG / ADA Compliance Issue from Malformed <UL> HTML

### ISSUE THREAD

**Issue by tonydjukic** [viljamis/ResponsiveSlides.js#418]
When building the code for the `$pager` variable on line 68 in `responsiveslides.js` the `<ul>` tag for the pager isn't closed properly.

You have:
```
$pager = $("<ul class='" + namespace + "_tabs " + namespaceIdx + "_tabs' />"),
```

Whereas I believe the correct code should be:
```
$pager = $("<ul class='" + namespace + "_tabs " + namespaceIdx + "_tabs'></ul>"),
```

The current code results in an empty `<ul></u>` being included immediately after the opening `<ul>` as follows...

```
<ul class="rslides_tabs rslides1_tabs">
    <ul></ul>
```

This triggers some WCAG alerts in different scanning utilities.
